examples/xgboost/xgboost_simple.py

```diff
@@ -1,65 +1,79 @@
-parameters.
-
-"""
-
-import numpy as np
-import sklearn.datasets
-import sklearn.metrics
-from sklearn.model_selection import train_test_split
-import xgboost as xgb
-
-import optuna
-
-
-# FYI: Objective functions can take additional arguments
-# (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).
-def objective(trial):
-    (data, target) = sklearn.datasets.load_breast_cancer(return_X_y=True)
-    train_x, valid_x, train_y, valid_y = train_test_split(data, target, test_size=0.25)
-    dtrain = xgb.DMatrix(train_x, label=train_y)
-    dvalid = xgb.DMatrix(valid_x, label=valid_y)
-
-    param = {
-        "verbosity": 0,
-        "objective": "binary:logistic",
-        "booster": trial.suggest_categorical("booster", ["gbtree", "gblinear", "dart"]),
-        "lambda": trial.suggest_float("lambda", 1e-8, 1.0, log=True),
-        "alpha": trial.suggest_float("alpha", 1e-8, 1.0, log=True),
-    }
-
-    if param["booster"] == "gbtree" or param["booster"] == "dart":
-        param["max_depth"] = trial.suggest_int("max_depth", 1, 9)
-        param["eta"] = trial.suggest_float("eta", 1e-8, 1.0, log=True)
-        param["gamma"] = trial.suggest_float("gamma", 1e-8, 1.0, log=True)
-        param["grow_policy"] = trial.suggest_categorical("grow_policy", ["depthwise", "lossguide"])
-    if param["booster"] == "dart":
-        param["sample_type"] = trial.suggest_categorical("sample_type", ["uniform", "weighted"])
-        param["normalize_type"] = trial.suggest_categorical("normalize_type", ["tree", "forest"])
-        param["rate_drop"] = trial.suggest_float("rate_drop", 1e-8, 1.0, log=True)
-        param["skip_drop"] = trial.suggest_float("skip_drop", 1e-8, 1.0, log=True)
-
-    bst = xgb.train(param, dtrain)
-    preds = bst.predict(dvalid)
-    pred_labels = np.rint(preds)
-    accuracy = sklearn.metrics.accuracy_score(valid_y, pred_labels)
-    return accuracy
-
-
-if __name__ == "__main__":
-    study = optuna.create_study(direction="maximize")
-    study.optimize(objective, n_trials=100, timeout=600)
-
-    print("Number of finished trials: ", len(study.trials))
-    print("Best trial:")
-    trial = study.best_trial
-
-    print("  Value: {}".format(trial.value))
-    print("  Params: ")
-    for key, value in trial.params.items():
-        print("    {}: {}".format(key, value))
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset
+using XGBoost.
+
+In this example, we optimize the validation accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+
+"""
+
+import numpy as np
+import sklearn.datasets
+import sklearn.metrics
+from sklearn.model_selection import train_test_split
+import xgboost as xgb
+
+import optuna
+
+
+# FYI: Objective functions can take additional arguments
+# (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).

```

Comment:
```suggestion
```

If we don't use additional arguments here, I don't think it's useful to mention it's possible.

Reply:
Thanks for your comment. I agree with you that the lines seem redundant related to this example. It was in the older example before I modified it, but anyway I'll remove that.